libs/wire-api/src/Wire/API/User/Scim.hs

```diff
@@ -353,17 +354,28 @@ data ValidScimId = ValidScimId
 
 instance Arbitrary ValidScimId where
   arbitrary = do
-    authInfo <- QC.arbitrary
-    -- TODO(fisx): delete qualifiers from name ID
+    authInfo <-
+      QC.arbitrary
+        & fmap (mapThere removeQualifiers)
+        & flip suchThat shortShowNameIDIsNotNothing
```

Comment:
`unsafeShowNameID` also works with qualifiers, it just adds the extra requirement that qualifiers must not create different overlapping scopes.

also, how can shortShowNameIDIsNotNothing be false on a return value of removeQualifiers?